.github/workflows/aws-deploy.yml

```diff
@@ -26,17 +33,17 @@ jobs:
         with:
           cdk_subcommand: 'synth'
           actions_comment: false
-          cdk_args: '--context env=dev --debug'
+          cdk_args: '--context env=${{ env.CONTEXT }} --debug'
 
       - name: Install dependencies
         run: pip install -r requirements.txt -r requirements-dev.txt
 
       - name: Run unit tests
         run: python -m pytest tests/ -s -v
 
-  cdk-deploy-dev:
+  cdk-deploy:
     runs-on: ubuntu-latest
-    if: ${{ github.ref == 'refs/heads/main' }}
+    if: ${{ github.ref == 'refs/heads/main' || github.ref == 'refs/heads/prod'}}
```

Comment:
our workflow is to create a `dev` and `prod` branch, not `main`.  should we maybe create a dev and prod branch for this repo?